lib/jake/array.ex

```diff
@@ -1,5 +1,99 @@
 defmodule Jake.Array do
-  def gen(_) do
-    StreamData.constant([])
+  @type_list [
+    %{"type" => "integer"},
+    %{"type" => "number"},
+    %{"type" => "boolean"},
+    %{"type" => "string"},
+    %{"type" => "null"},
+    nil
+  ]
+
+  @min_items 0
+
+  @max_items 1000
+
+  def gen_array(map, type), do: arraytype(map, map["enum"], map["items"])
+
+  def arraytype(map, enum, items) when enum != nil, do: Jake.gen_enum(enum, "array")
```

Comment:
could this happen? won't the enum get handle at the Jake module?

Reply:
Consider schema,
    ``{"type": "integer", "enum": [10,12, null]}``
It will first check type and enter into Jake.Number module. If it finds, enum list there then it will be handled by it.

If schema is ``{"enum": [10,12, null]}``, without mentioning type then it will be handled by Jake module.

Reply:
but the pattern match for enum happens early? so this will never get executed?

```
  def gen_all(map, enum, type) when enum != nil, do: gen_enum(enum, type)
```

Reply:
btw, is it really necessary to handle enum in all type-specific generator? anyway it's going to call the root module? we could instead just remove the enum handling from all the type-specific generators?

Reply:
I was also thinking the same. I'll check it out!

Reply:
> we could instead just remove the enum handling from all the type-specific generators?

It won't handle case like ``{"type":["integer", "string"], "enum": [1,2,3,"hello", null, true]}``. Otherwise all other cases will work. In order to accept above case, we'll need to change/override ``Jake.gen_enum()`` method to handle list of types. 

I'll see, if it can be done without complicating things.